Dev_Planning/verify_p0_smoke.py

```
@@ -0,0 +1,110 @@
+        expected = sorted(files.keys(), key=lambda s: s.casefold())
+        assert merged_paths == expected, (
+            "Concatenated header order mismatch:\n"
+            + "\n".join(["got:    " + p for p in merged_paths])
+            + "\n"
+            + "\n".join(["expect: " + p for p in expected])
+        )
+
+        # Atomic write failure: ensure no final file created on exception
+        final = out_dir / "should_not_exist.txt"
+        try:
+            loader._atomic_write_text(final, lambda fh: (_ for _ in ()).throw(RuntimeError("boom")))
```

Ruff (0.14.4)

Unused lambda argument: `fh`

(ARG005)